config/default/translations/messages-en.properties

```diff
@@ -0,0 +1,1648 @@
+Contact = Contact
+From = From
+Clinic = Household
+Clinics = Families
+Village\ Name = Town
+Clinic\ Contact\ Name = Name
+Clinic\ Contact\ Phone = Phone number
+RC\ Code = Code
+Health\ Centers = Health centers
+Health\ Center = Health center
```

Comment:
Are you tracking work to add translations for this body of work?

Reply:
Sorry, I could not understand it. Most of these translations are removed now and I am keeping only the ones that are either new or modified ones.

Reply:
Is the CHT Reference App going to support French, Hindi, and other languages?